### ISSUE TITLE
Anchor layer

### ISSUE THREAD

**Issue by 0x00b1** [broadinstitute/keras-rcnn#8]
It should produce anchors from ground truth bounding boxes.

This is the first of two layers needed by the regional proposal network (RPN) layers.

**Comment by 0x00b1** [comment]
We’ve decided to replace the Anchor layer by generating anchors inside the RPN loss function. 

cc: @JihongJu and @jhung0
